bio_hansel/main.py

```diff
@@ -224,11 +225,15 @@ def main():
 
     dfs = [df for st, df in subtype_results]  # type: List[pd.DataFrame]
     dfsummary = pd.DataFrame([attr.asdict(st) for st, df in subtype_results])
+
     dfsummary = dfsummary[SUBTYPE_SUMMARY_COLS]
 
     if dfsummary['avg_tile_coverage'].isnull().all():
         dfsummary = dfsummary.drop(labels='avg_tile_coverage', axis=1)
 
+    if dfsummary['subtype'].isnull().any():
```

Comment:
Is it possible to drop this check and just `fillna`? I'm assuming the results from doing the check and then performing `fillna` on DFs with and without NA/null values would be the same.

Reply:
That does work. I've made the requested change and will look at the other one

Thanks